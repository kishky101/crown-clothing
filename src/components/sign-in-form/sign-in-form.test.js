import renderer from 'react-test-renderer';
import SignInForm from './sign-in-form.component';
import { MockStore } from '../../utils/test/setup';
import { BrowserRouter as Router } from 'react-router-dom';

test('creating sign-in-form snapshot', () => {
    const signinform = renderer.create(
        <MockStore>
            <Router>
                <SignInForm />
            </Router>
        </MockStore>
    ).toJSON;
    expect(signinform).toMatchSnapshot();
})