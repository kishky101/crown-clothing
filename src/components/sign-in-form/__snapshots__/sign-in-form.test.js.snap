// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating sign-in-form snapshot 1`] = `[Function]`;
